flink/sbt/sbt.rst

```diff
@@ -0,0 +1,57 @@
+===============================
+Using SBT to create a Flink job
+===============================
+
+Motivation
+----------
+
+Maven is a widely accepted build tool for projects that target the Java platform. The Radicalbit Fast Data Distribution provides an archetype to kickstart your development projects using Maven, but Scala developers may find more compelling to use the Maven alternative of choice for the Scala ecosystem, SBT (Simple Build Tool).
+
+Basic 'archetype'
+-----------------
+
+Currently, SBT does not provide built-in capabilities to scaffold a project like Maven archetypes do.
+As such, some manual work is required to initialize the build definition properly.
```

Comment:
Ok, I'll add a link to the source file since the project is shared among several examples. I'll open an issue to update it.